### PR TITLE
Fixes #24757 - fix audit searching by user

### PR DESCRIPTION
**Pull request by ares**
to test:

1) run foreman + katello in production mode
2) go to audits
3) try searching by query "type = user"

the problem is that Foreman creates the list dynamically but "ignores" STI classes, user in core is not STI-ed but Katello changes this, there it has also to add it to the explicit mapping list

### CODE REVIEW

**Comment by jlsherrill** [comment]
Rubocop error: /var/lib/workspace/workspace/katello-pr-test/app/models/katello/concerns/audit_search.rb:14:1: C: Layout/TrailingBlankLines: 1 trailing blank lines detected.

File: db/seeds.d/102-organizations.rb [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-end
+#if ENV['SEED_ORGANIZATION']
+#  if Setting['db_pending_seed']
+#    admin = User.where(:login => ENV['SEED_ADMIN_USER'].present? ? ENV['SEED_ADMIN_USER'] : 'admin').first
+#    if admin && admin.default_organization.nil?
+#      admin.default_organization = Organization.find_by(:name => ENV['SEED_ORGANIZATION'])
+#      admin.save!
+#    end
+#  end
+#end
```
Comment (bastilian) [inline db/seeds.d/102-organizations.rb]: Is this change necessary?

Comment (ares) [reply db/seeds.d/102-organizations.rb]: of course not, `git commit -a --amend` :-) thanks for heads up, removed from the commit

**Comment by jlsherrill** [comment]
[test katello]
